controller/articles.controller.js

```diff
@@ -11,8 +11,9 @@ exports.getAllArticles = (req, res, next) => {
 
 exports.getArticlesById = (req, res, next) => {
   const { article_id } = req.params;
+  const { include } = req.query;
```

Comment:
This is not needed as we should not be dealy with a query for this comment_count property